src/main/java/com/openshift/internal/restclient/model/ReplicationController.java

```diff
@@ -364,4 +366,24 @@ public void addVolume(IVolumeSource volumeSource) {
 		}
 		volList.add(ModelNode.fromJSONString(volumeSource.toJSONString()));
 	}
+
+    @Override
+    public void setServiceAccount(String serviceAccount) {
```

Comment:
Why do we have these getters and setters?  I dont see that as an available part of the pod spec